patches/@hyperledger__anoncreds-react-native@0.2.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ios/Anoncreds.mm b/ios/Anoncreds.mm
+index 6007a03d12a88cad8d64eedd10c9075ccaf4b84f..1a0a66c6e0e701c169aaa8476bf29fdae7a62495 100644
+--- a/ios/Anoncreds.mm
++++ b/ios/Anoncreds.mm
+@@ -3,7 +3,7 @@
+ #import <React/RCTUtils.h>
+ #import <ReactCommon/RCTTurboModule.h>
+ 
+-#import "turboModuleUtility.h"
++#import <anoncreds/turboModuleUtility.h>
```

Comment:
Is this because of static? Can we add this to the library itself by detecting the mode with an #if or something (for later, but so we can use the libraries without patches)

Reply:
I am not really sure why we had to change this. It will be removed in the new architecture as well.